lib/polyn/utils/hash.rb

```diff
@@ -29,8 +29,11 @@ module Hash
       #
       # @return [::Hash] The symbolized hash.
       def self.deep_symbolize_keys(hash)
```

Comment:
Why can't we just use the ruby built in hash.deep_symbolize_keys! method?

Reply:
I'm not finding that as a built-in method? Is that a rails thing?

Reply:
Ya, looks like it comes from active_support https://www.rubydoc.info/gems/activesupport/4.0.0/Hash:deep_symbolize_keys

Reply:
Ah, the trick of only working in a ruby on rails app and not a pure ruby app.